### PR TITLE
HELM-217/HELM-218/HELM-227: Integration of converted Antora documentation

### PR DESCRIPTION
**Pull request by indigo423**
# Pull Request Check Sheet

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: the subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new feature, is there documentation?
* [x] If this is a new feature or substantial change, are there tests that cover it?

We have migrated all the Helm documentation from Asciibinder to Antora 🎉. The build pipeline uses the official Antora docker image to generate the HTML doc output and we can publish it now with Netlify when we run on master. The PR incorporates also all the work migrating the content from AsciiBinder to Antora.

* JIRA (Issue Tracker): http://issues.opennms.org/browse/HELM-217
* JIRA (Issue Tracker): http://issues.opennms.org/browse/HELM-218
* JIRA (Issue Tracker): http://issues.opennms.org/browse/HELM-227

### CODE REVIEW

**Comment by Bonrob2** [comment]
As a note, this documentation includes the updated text for HELM-207 and 206 on entities datasource and filter panel.

File: docs/modules/about/pages/legal_notice.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[welcome-legal-notice]]
 = Legal Notice
-{product-author}
-{product-version}
+{}
```
Comment (RangerRick) [inline docs/modules/about/pages/legal_notice.adoc]: I don't know how Antora works but is there a reason for this empty `{}` ?

Comment (Bonrob2) [reply docs/modules/about/pages/legal_notice.adoc]: @RangerRick, my mistake. I did a global search to remove the "product-author" variable, and missed the {}.

File: docs/modules/about/pages/introduction.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
+:imagesdir: ../assets/images
```
Comment (indigo423) [inline docs/modules/about/pages/introduction.adoc]: @Bonrob2 I've just noticed, the `assets/images` directory is in `pages`. In the other section directories is it a directory higher. To make it work here it must be `assets/images`.

Comment (Bonrob2) [reply docs/modules/about/pages/introduction.adoc]: Ugh. Yes. You are right. I will fix that.

**Comment by RangerRick** [comment]
Is this mergeable? I'd like to do a Helm 5 release but I wanted to get all outstanding PRs wrapped up first.

**Comment by indigo423** [comment]
To be honest, I would not recommend blocking the 5 release with this PR. So far I can see we have a few organizational things left. First thing, as soon we merge this PR

* the docs get published to an intermediate site and just for version 5.0.0
* We need to updated our docs.opennms.org
* We don't have a version history with Antora. All previous version where generated with AsciiBinder. We need a place for the old AsciiBinder docs
* We have to define a release process for Antora tags / branches, as far I know, we have just a manual release process

Just my quick 0,02 cents

**Comment by Bonrob2** [comment]
Agreed. Ronny, what do you think the timeline around that would be? I'd like to get the content from HELM 206&207 (entities datasource and filter panel) in an upcoming release. I have a separate branch for that that I didn't merge this time.

**Pull request merged**